dao/src/main/java/com/epam/esm/criteria/tags/AbstractTagCriteria.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.esm.criteria.tags;
+
+import com.epam.esm.criteria.Criteria;
+import com.epam.esm.entity.GiftCertificate;
+import com.epam.esm.entity.Tag;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+
+/**
+ * Abstract Criteria class with common fields.
+ *
+ * @since 1.0
+ */
+public abstract class AbstractTagCriteria implements Criteria<Tag> {
+
+    protected final static String TABLE_NAME = "tag";
```

Comment:
@gkttk Could you move all constants to the utils class

Reply:
commit 5167a06